src/lsstseries/analysis/structure_function/base_calculator.py

```diff
@@ -14,9 +14,9 @@ class StructureFunctionCalculator(ABC):
 
     def __init__(
         self,
-        time: List[float],
-        flux: List[float],
-        err: List[float],
+        time: List[List[float]],
+        flux: List[List[float]],
+        err: List[List[float]],
```

Comment:
I just wanted to be more explicit here, since we do expect that these inputs will be 2d arrays.

Reply:
Would these be more efficient as numpy arrays? (Not necessarily in scope of this change).

Reply:
This is good call out, they actually are passed in as 2d numpy arrays. And it prompts me to look up how to do type hints for numpy arrays. Looks like this is at least one approach: https://numpy.org/doc/stable/reference/typing.html#module-numpy.typing

Reply:
Done